content/client-lib-development-guide/features.textile

```diff
@@ -1336,6 +1339,32 @@ h4. ChannelProperties
 * @(CP2)@ The attributes of @ChannelProperties@ consist of:
 ** @(CP2a)@ @attachSerial@ string - contains the last @channelSerial@ received in an @ATTACHED@ @ProtocolMesage@ for the channel, see "RTL15a":#RTL15a
 
+h4. ChannelDetails
+
+* @(CHD1)@ @ChannelDetails@ is a type that represents information for a channel including channelID, name, status and occupancy 
```

Comment:
The API returns a `channelId` field (note the lower case `d`), I think we should be consistent and refer to it as `channelId` here in the document too.

Reply:
Good spot, I've pushed this change.